vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/project/info/ProjectDetailsComponent.java

```diff
@@ -244,9 +237,9 @@ private void generateExperimentInformationTags(List<Experiment> experiments) {
     analytesField.add(analysisSet.stream().map(Tag::new).collect(Collectors.toList()));
   }
 
-  private Div generatePersonReference(PersonReference personReference) {
-    Span nameSpan = new Span(personReference.fullName());
-    Span emailSpan = new Span(personReference.emailAddress());
+  private Div generatePersonReference(Contact contact) {
+    Span nameSpan = new Span(contact.fullName());
+    Span emailSpan = new Span(contact.emailAddress());
```

Comment:
Outdated method and variable names since personReference does not exist anymore in this PR